ontology.ttl

```diff
@@ -1014,6 +1014,21 @@ oe:dos_heeftBijlage a owl:ObjectProperty;
     Deze relatie geeft aan dat B een bestand is dat toegevoegd werd aan A.
     """@nl.
 
+oe:dos_heeftBriefBeslissing a owl:ObjectProperty;
```

Comment:
Klinkt zinnig. Je zou er een apart type van kunnen maken, met een property "beslissingstype", maar dat is denk ik momenteel geen vereiste voor de ontology?

Reply:
Nee, met deze oplossing kunnen we al verder.